api-server/src/server/boot/challenge.js

```diff
@@ -121,6 +127,10 @@ const savableChallenges = getChallenges()
   .filter(challenge => challenge.challengeType === 14)
   .map(challenge => challenge.id);
 
+const msTrophyChallenges = getChallenges()
+  .filter(challenge => challenge.challengeType === 18)
+  .map(({ id, msTrophyId }) => ({ id, msTrophyId }));
```

Comment:
Shouldn't you be importing the `challengeType` here?

```ts
import { challengeType } from '../../../../config/challenge-types.ts';

// omitted for brevity
```

Then you'd use the `const` value?

```suggestion
const msTrophyChallenges = getChallenges()
  .filter(challenge => challenge.challengeType === challengeType.msTrophy)
  .map(({ id, msTrophyId }) => ({ id, msTrophyId }));
```

Reply:
Yup, added ✅ 

I will make an issue to fix the rest of the spots a magic number is used in that file.